lib/user.rb

```diff
@@ -0,0 +1,35 @@
+#User.rb
+
+require 'bcrypt'
+require 'data_mapper'
+
+class User
+  attr_accessor :password, :password_confirm
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String, length: 300
+  property :username, String, length: 300, unique: true, message: "This username is taken, think harder"
```

Comment:
Metrics/LineLength: Line is too long. [104/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)